build_and_release.sh

```diff
@@ -5,6 +5,43 @@ set -euo pipefail
 # shellcheck source=./yb-thirdparty-common.sh
 . "${BASH_SOURCE%/*}/yb-thirdparty-common.sh"
 
+install_cmake_on_macos() {
```

Comment:
This seems to be done  regardless of what version currently exists. Does this work on mac dev system if run more than once?

Does it work in yugabyte-db  tree if dev decides not to download thirdparty, but build it? And what if they don't have sudo access?

Reply:
This script is only invoked in the CI/CD path, not as part of YugabyteDB build.

Reply:
Actually I've removed this, as CMake 3.19.2 and onwards works.